ta4j-core/src/main/java/org/ta4j/core/indicators/DistanceFromMAIndicator.java

```diff
@@ -23,15 +23,14 @@
  */
 package org.ta4j.core.indicators;
 
-import org.ta4j.core.Bar;
-import org.ta4j.core.BarSeries;
-import org.ta4j.core.num.Num;
-
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.HashSet;
 import java.util.Set;
 
+import org.ta4j.core.Bar;
+import org.ta4j.core.BarSeries;
+import org.ta4j.core.num.Num;
+
```

Comment:
is all this swapping of imports done by your IDE or by the formatter plugin?

Reply:
I use Eclipse and by using `Source->Organize Imports` you can improve the ordering of the import statements. This is done automatically and it's recommended. I don't know how to do that with formatter plugin.